### ISSUE TITLE
nvda doesn't read the multi line editbox contents in AWS's services such as amplify, lambda, 

### ISSUE THREAD

**Issue by alexandretoco** [nvaccess/nvda#12564]
### Steps to reproduce:
Follow the tutorial to access aws lambda and do the steps to open aws iam manager.
https://aws.amazon.com/pt/getting-started/hands-on/build-web-app-s3-lambda-api-gateway-dynamodb/module-four/

The tutorial gives a piece of code that has to be placed in a multi-line editbox. Here I can't read the content in focus mode.
### Actual behavior:
nvda doesn't read the multi-line editbox content in focus mode.
### Expected behavior:
It should read the editbox content and allow the user to edit it
#### NVDA installed/portable/running from source:
installed
#### NVDA version:
2021.1 beta 3
#### Windows version:
windows 10
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
nvda 2020.4 also doesn't read these editboxes

**Comment by feerrenrut** [comment]
I have looked at the link, and couldn't find the sample you are talking about. Please create a minimal test case, either zipped and attached to this issue or on codePen or similar service.

**Comment by zersiax** [comment]
@alexandretoco  If you paste stuff into that box, see if the code shows up in browse mode right below the edit field. If so, you just stubmbled across one of the websites that use CodeMirror or similar that cause NVDA to be unable to read the contents of the edit field properly. CodeMirror at least has a beta up that fixes this, but if there are other editors that work along the same lines you'd get this behavior.

**Comment by feerrenrut** [comment]
From @zersiax's comment https://github.com/nvaccess/nvda/issues/12564#issuecomment-882482168. It seems like this is a web authoring error. Please contact AWS support. I'm going to close this issue.
